tasks/rh-sign-image/tests/test-rh-sign-image-skip-signing-when-signed.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-rh-sign-image-skip-signing-when-signed
+spec:
+  description: Test skip signing when already signed for the rh-sign-image task
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              cat > $(workspaces.data.path)/snapshot_spec.json << EOF
+              {
+                "application": "myapp",
+                "components": [
+                  {
+                    "name": "comp0",
+                    "source": {
+                      "git": {
+                        "revision": "deadbeef"
+                      }
+                    },
+                    "containerImage": "registry.io/image0@sha256:0000",
+                    "repository": "quay.io/redhat-prod/myproduct----myrepo",
+                    "rh-registry-repo": "registry.redhat.io/myproduct/myrepo",
```

Comment:
You should use some string that makes it obvious that this is the scenario where it's already signed. That will also make it obvious in the mock.

Reply:
Actually, I think that's also what broke the other test, because it uses the same pull spec.

Reply:
Oh, yes, I got it.
Thanks!

Reply:
Fixed!